drivers/adc/adc_demo/adc_demo.c

```diff
@@ -41,11 +41,34 @@
 /***************************** Include Files **********************************/
 /******************************************************************************/
 
-#include <stdlib.h>
-#include "adc_demo.h"
+#include <stdio.h>
+#include <string.h>
+#include <inttypes.h>
+#include "iio_types.h"
+#include "iio_adc_demo.h"
```

Comment:
iio_adc_demo.h shouldn't be needed here (the driver doesn't need to know about iio)